### PR TITLE
feat: add custom labels to report 

### LINKED ISSUE #943: Add additional labels or annotations on CRDs like VulnerabilityReport
Hi Trivy-operator community members, 

I want it ask if it is possible to add additional labels or annotations in the VulnerabilityReport crd?

Currently this is the part of VulnerabilityReport yaml file I get. I want to customize it to add some information like the Kubernetes cluster name, dev or live environment, etc. I am using Helm chart to deploy. I have tried all label and annotation key value pairs in current helm chart but they don't change the VulnerabilityReport crd yaml file.

> apiVersion: aquasecurity.github.io/v1alpha1
kind: VulnerabilityReport
metadata:
  annotations:
    trivy-operator.aquasecurity.github.io/report-ttl: 24h0m0s
  creationTimestamp: '2023-02-09T12:38:33Z'
  generation: 4
  labels:
    resource-spec-hash: cc65f4d64
    trivy-operator.container.name: kube-proxy
    trivy-operator.resource.kind: DaemonSet
    trivy-operator.resource.name: kube-proxy
    trivy-operator.resource.namespace: kube-system
  managedFields: ...

Any suggestions and improvements are appreciated.
Best regards,
Weibo

### PR DESCRIPTION
**Pull request by chen-keinan**
## Description
add custom labels to report 

## Related issues
- Close #943

## Checklist
- [x] I've read the [guidelines for contributing](https://github.com/aquasecurity/trivy-operator/blob/main/CONTRIBUTING.md) to this repository.
- [x] I've added tests that prove my fix is effective or that my feature works.
- [x] I've updated the [documentation](https://github.com/aquasecurity/trivy-operator/tree/main/docs) with the relevant information (if needed).
- [x] I've added usage information (if the PR introduces new options)
- [x] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: pkg/trivyoperator/config.go [hunk 1]
```diff
@@ -72,6 +72,7 @@ const (
 	KeyMetricsResourceLabelsPrefix         = "metrics.resourceLabelsPrefix"
 	KeyTrivyServerURL                      = "trivy.serverURL"
 	KeyNodeCollectorImageRef               = "node.collector.imageRef"
+	KeyAdditionalReportLables              = ""
```
Comment (SamarSidharth) [inline pkg/trivyoperator/config.go:75]: I think we need to have a corresponding field in  the helm config map template and then it should be referenced here.

Comment (chen-keinan) [reply pkg/trivyoperator/config.go:75]: @SamarSidharth defiantly :) the pr its still WIP

File: deploy/helm/README.md [hunk 2]
```diff
@@ -87,6 +87,7 @@ The values of the `OPERATOR_NAMESPACE` and `OPERATOR_TARGET_NAMESPACES` determin
...
 | `scanJob.nodeSelector`| N/A| JSON representation of the [nodeSelector] to be applied to the scanner pods so that they can run on nodes with matching labels. Example: `'{"example.com/node-type":"worker", "cpu-type": "sandylake"}'`                                                                                        |
 | `scanJob.automountServiceAccountToken`         | `"false"`   | the flag to enable automount for service account token on scan job. Set `"true"` to enable.                                                                                                                                                                                                     |
+| `report.additionalLables`         | `""`   |  additionalReportLables comma-separated representation of the labels which the user wants the scanner pods to be labeled with. Example: `foo=bar,env=stage` will labeled the reports with the labels `foo: bar` and `env: stage`                                                                                                                                                                                                     |
```
Comment (chap-dr) [inline deploy/helm/README.md:90]: Should `scanner pods` perhaps be replaced with something linke `reports` ? (in this and the other 3 occurences)

Comment (daniel-laszlo) [inline deploy/helm/README.md:90]: typo: `additionalLables` -> `additionalLabels`

Comment (chen-keinan) [reply deploy/helm/README.md:90]: @daniel-laszlo can you make it in a separate PR. as its already merged

**Pull request merged**